src/ip_grid_descriptor_mod.f90

```diff
@@ -49,6 +70,10 @@ function init_grib1_descriptor(gds) result(desc)
 
   end function init_grib1_descriptor
 
+  !> @brief Initialize grib-2 descriptor from integer grid definition template
```

Comment:
Punctuate. Also, for functions/subroutines you don't need to use the @breif tag, the first sentence will be used as the brief description, if AUTOBRIEF is turned on in Doxyfile, as it should be.